azure-pipelines.yml

```diff
@@ -0,0 +1,67 @@
+# `SkipCucumberJob` variable points to the matrix case that will skip performing of cucumber tests
+variables:
+  SkipCucumberJob: 'Mojave-Xcode-11.0'
+
+jobs:
+
+- job:
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+  pool:
+    vmImage: $(IMAGE_POOL)
+
+  steps:
+
+  - script: |
+      set -e
+      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+      echo "Xcode version: $(xcrun xcodebuild -version)"
+      echo "Xcode path: $(xcrun -f xcodebuild)"
+      echo "Xcode select path: $(xcode-select --print-path)"
+    displayName: "Select Xcode $(XCODE_VERSION)"
+
+
+  - script: |
+      set -e
+      ruby -v
+      echo -e "install: --no-document --env-shebang\nupdate:  --no-document --env-shebang" > ~/.gemrc
+      bundle install
+    displayName: "Prepare Ruby Environment"
+
+  - script: |
+      set -e
+      git clone https://$(GitHubAccessToken)@github.com/xamarinhq/calabash-codesign.git
+      calabash-codesign/apple/create-keychain.sh
+    displayName: "Download and install keychain"
+
+  - script: make app-agent
+    displayName: "Make app-agent"
+
+  - script: make ipa-agent
+    displayName: "Make ipa-agent"
+
+  - script: make test-ipa
+    displayName: "Make test-ipa"
+
+  - script: make test-app
+    displayName: "Make test-app"
+
+  - script: make unit-tests
+    displayName: "Make unit-tests"
+
+  - script: bundle exec bin/ci/cucumber.rb
+    displayName: "exec cucmber"
+    condition: and(succeeded(), ne(variables['Agent.JobName'], variables['SkipCucumberJob']))
```

Comment:
May be condition `eq(variable['XCODE_VERSION'], '11')` will be clearer?

Reply:
Done